firmware/applications/SignBuddy/imu.c

```diff
@@ -0,0 +1,119 @@
+#include "imu.h"
+
+#include "board.h"
+#include "common.h"
+#include "i2c.h"
+#include "logger.h"
+#include "bno055.h"
+
+#define PROCESS_PERIOD_MS    1000
+
+typedef struct {
+  uint32_t                last_ticks;
+  i2c_t                   i2c_instance;
+  struct   bno055_t       bno055;
+  struct   bno055_accel_t bno055_accel_xyz;
+  struct   bno055_mag_t   bno055_mag_xyz;
+  struct   bno055_gyro_t  bno055_gyro_xyz;
+} state_t;
+
+static state_t s;
```

Comment:
Rename to state

Reply:
you mean rename s to state (s -> state) or also the struct member names (bno055_gyro_xyz -> state_bno055_gyro_xyz)?

Reply:
I meant change `s` to `state`